server/routes/activities/create-an-activity/handlers/sessionTimes.ts

```diff
@@ -19,18 +19,18 @@ type SessionSlot = {
   isFirst: boolean
```

Comment:
Looks like day is being shown for all slots for that day?

In which case you can remove this flag.